shoov/modules/custom/shoov_restful/plugins/restful/node/ui_builds/1.0/ShoovUiBuildsResource.class.php

```diff
@@ -31,7 +31,7 @@ public function publicFieldsInfo() {
       'resource' => array(
         'repository' => array(
           'name' => 'repositories',
-          'full_view' => FALSE,
+          'full_view' => TRUE,
```

Comment:
I ran UI regression tests on the `test-example` repo and the tests got pushed into the list as expected, Is this what you mean ?